tests/integrations/object_detection/test_object_detection.py

```diff
@@ -0,0 +1,242 @@
+        onnx_file = os.path.join(
+            os.path.dirname(export_args.run_args.weights), "last.onnx"
+        )
+
+        assert os.path.isfile(onnx_file)
+
+        model = onnx.load(onnx_file)
+        onnx.checker.check_model(model)
+
+    @skip_inactive_stage
+    def test_export_target_model(self, integration_manager):
+        # If target model provided in target_args, test that they have similar
+        # ort output
+        manager = integration_manager
+        export_args = manager.configs["export"]
+        target_model_path = export_args.test_args.get("target_model")
+
+        if not target_model_path:
+            pytest.skip("No target model provided")
+
+        export_model_path = os.path.join(
+            os.path.dirname(export_args.run_args.weights), "last.onnx"
+        )
+
+        # Downloads model if zoo stubs and additionally tests that it can be loaded
+        _, *_ = load_checkpoint(
+            type_="val", weights=target_model_path, device=torch.device("cpu")
+        )
+
+        _test_model_op_counts(export_model_path, target_model_path)
+
+        compare_outputs = export_args.test_args.get("compare_outputs", True)
+        if isinstance(compare_outputs, str) and (
+            compare_outputs.lower() in ["none", "False"]
+        ):
+            compare_outputs = False
+        if compare_outputs:
+            _test_model_inputs_outputs(export_model_path, target_model_path)
+
+
+_TEST_OPS = {
+    "MatMul",
+    "Gemm",
+    "Conv",
+    "MatMulInteger",
+    "ConvInteger",
+    "QLinearMatMul",
+    "QLinearConv",
+}
+
+
+def _test_model_op_counts(model_path_a, model_path_b):
+
+    model_a = onnx.load(model_path_a)
+    model_b = onnx.load(model_path_b)
+
+    def _get_model_op_counts(model):
+        op_counts = defaultdict(int)
+        for node in model.graph.node:
+            if node.op_type in _TEST_OPS:
+                op_counts[node.op_type] += 1
+        return op_counts
+
+    op_counts_a = _get_model_op_counts(model_a)
+    op_counts_b = _get_model_op_counts(model_b)
+
+    assert len(op_counts_a) > 0
+    assert len(op_counts_a) == len(op_counts_b)
+
+    for op, count_a in op_counts_a.items():
+        assert op in op_counts_b
+        assert count_a == op_counts_b[op]
+
+
+def _test_model_inputs_outputs(model_path_a, model_path_b):
+    # compare export and target graphs and build fake data
+    model_a = onnx.load(model_path_a)
+    model_b = onnx.load(model_path_b)
+    assert len(model_a.graph.input) == len(model_b.graph.input)
+    assert len(model_a.graph.output) == len(model_b.graph.output)
+
+    sample_input = {}
+    output_names = []
+
+    for input_a, input_b in zip(model_a.graph.input, model_b.graph.input):
+        assert input_a.name == input_b.name
+        input_a_shape = _get_tensor_shape(input_a)
+        assert input_a_shape == _get_tensor_shape(input_b)
+        sample_input[input_a.name] = numpy.random.randn(*input_a_shape).astype(
+            numpy.float32
```

Comment:
quantized YOLOv5 graphs will have uint8 inputs. need to account for this (can look at input layer dtype)